prometheus-alertmanager.yaml

```diff
@@ -45,6 +44,35 @@ pipeline:
 
   - uses: strip
 
+subpackages:
+  - name: ${{package.name}}-bitnami-compat
+    dependencies:
+    description: "compat package with bitnami/alertmanager image"
+    pipeline:
+      - runs: |
+          mkdir -p ${{targets.contextdir}}/opt/bitnami/alertmanager/bin/
+          mkdir -p ${{targets.contextdir}}/opt/bitnami/alertmanager/conf/
+          mkdir -p ${{targets.contextdir}}/opt/bitnami/alertmanager/data/
+          mkdir -p ${{targets.contextdir}}/etc/alertmanager
+          mkdir -p ${{targets.contextdir}}/alertmanager
+          chmod g+rwX ${{targets.contextdir}}/opt/bitnami/alertmanager/data/
+          ln -sf /usr/bin/alertmanager ${{targets.contextdir}}/opt/bitnami/alertmanager/bin/alertmanager
+          ln -sf /usr/bin/amtool ${{targets.contextdir}}/opt/bitnami/alertmanager/bin/amtool
+          ln -sf /etc/alertmanager/alertmanager.yml ${{targets.contextdir}}/opt/bitnami/alertmanager/conf/config.yml
+          ln -sf /opt/bitnami/alertmanager/conf ${{targets.contextdir}}/etc/alertmanager
+          ln -sf /opt/bitnami/alertmanager/data ${{targets.contextdir}}/alertmanager
+    test:
+      environment:
+        contents:
+          packages:
+            - ${{package.name}}
```

Comment:
We shouldn't add this as melange populates it automatically.
```suggestion
```

Reply:
thank you for looking into it, given we're invoking the binary, including the main package in here is required. When we do sub-package test, it doesn't pull in the main package so we'll have to be explicit here.

Reply:
Oh, right :D